epregressions/tests/diffs/test_table_diff.py

```diff
@@ -409,3 +409,24 @@ def test_nbsp(self):
         self.assertEqual(0, response[6])  # size errors
         self.assertEqual(0, response[7])  # in file 2 but not in file 1
         self.assertEqual(0, response[8])  # in file 1 but not in file 2
+
+    def test_ignore_version_diff(self):
```

Comment:
New unit test exercising the table diff changes.

Reply:
I must be missing something. From the 2 htm files above it looks like the versions are the same. Copied here to compare.

    <p>Program Version:<b>EnergyPlus, Version 9.0.1-a7c9cc14ce, YMD=2018.11.20 17:26</b></p>
    <p>Program Version:<b>EnergyPlus, Version 9.0.1-a7c9cc14ce, YMD=2018.11.20 17:26</b></p>

Reply:
It's the **other** version table:

Base:
```html
<!-- FullName:Initialization Summary_Entire Facility_Version-->
<table border="1" cellpadding="4" cellspacing="0">
  <tr><td></td>
    <td align="right">Version ID</td>
  </tr>
  <tr>
    <td align="right">1</td>
    <td align="right">9.6</td>
  </tr>
</table>
```

Mod:
```html
<!-- FullName:Initialization Summary_Entire Facility_Version-->
<table border="1" cellpadding="4" cellspacing="0">
  <tr><td></td>
    <td align="right">Version ID</td>
  </tr>
  <tr>
    <td align="right">1</td>
    <td align="right">22.1</td>
  </tr>
</table>
```